tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -163,6 +163,7 @@ CREATE MATERIALIZED VIEW max_mat_view_timestamp
         GROUP BY 1 WITH NO DATA;
 SELECT add_continuous_aggregate_policy('max_mat_view_timestamp', '10 day', '1 h'::interval , '1 h'::interval) as job_id \gset
 CALL run_job(:job_id);
+NOTICE:  continuous aggregate "max_mat_view_timestamp" is already up-to-date
```

Comment:
Unrelated change to the PR. IMHO it should be handled separately, since it is unclear what the reason for this sudden problem.

Reply:
Tests won't pass if this line is not here so how can it be handled separately?

Reply:
It is really weird that the notice disappeared from the test.
handling separately: https://github.com/timescale/timescaledb/pull/2475

Reply:
Ok. Will wait for you to push that one then.

Reply:
Seems you closed that issue without pushing, so I should probably push this even though it isn't related to the change.

Reply:
I block your PR, this line should not be pushed. The notice will likely disappear tomorrow, since it is a flaky test. See https://github.com/timescale/timescaledb/pull/2477 as a temporal solution for now.

Reply:
Then I'll be waiting for you to say when I can remove this line.